src/Dfc.CourseDirectory.Core/DataManagement/FileUploadProcessor.cs

```diff
@@ -185,6 +187,32 @@ private void CheckStreamIsProcessable(Stream stream)
             }
         }
 
+        public async Task<bool> DeleteVenueUploadRow(Guid venueUploadId, int rowNumber)
```

Comment:
Would be good to make this `DeleteVenueUploadRowForProvider(Guid providerId, int rowNumber)` to align with other methods.

Reply:
done